docker/maintenance/cronjob-template.yaml

```diff
@@ -34,9 +35,11 @@ spec:
               limits:
                 cpu: "6"
                 memory: 1200Mi
+                ephemeral-storage: "400Gi"
               requests:
                 cpu: "2"
                 memory: 800Mi
+                ephemeral-storage: "100Gi"
```

Comment:
Just FYI:
1. `ephemeral-storage` resource is currently not working on our clusters
2. on the other hand your numbers are overoptimistic, how do you think we have hosts with spare 100-400Gi free space? On average, hosts have ~300GB for _all_ workloads data, images, etc. So in this case you would use one node exclusively only for your workload?

Reply:
I could but the dumps are quite large. I'm working on splitting this into smaller chunks and I will tune those when I know how much I need. This is quite temporary and should be removed in few days.